.rubocop_todo.yml

```diff
@@ -0,0 +1,23 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 99999`
+# on 2023-05-12 18:20:53 UTC using RuboCop version 1.50.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 9
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: have_no, not_to
+Capybara/NegationMatcher:
+  Exclude:
+    - 'spec/system/demo_splash_spec.rb'
+
+# Offense count: 10
+# Configuration parameters: EnforcedStyle, AllowedGems, Include.
+# SupportedStyles: Gemfile, gems.rb, gemspec
+# Include: **/*.gemspec, **/Gemfile, **/gems.rb
+Gemspec/DevelopmentDependencies:
+  Exclude:
+    - 'demo_mode.gemspec'
```

Comment:
preferring not to have `add_development_dependency` in the gemspec is news to me. we should probably configure this in `betterlint` as well